server/routes/check-your-details.js

```diff
@@ -159,12 +98,7 @@ module.exports = [
           try {
             const referer = request.headers ? request.headers.referer : undefined
             // TODO - reinstate this request
```

Comment:
can remove the comment now, otherwise it'll confuse

Reply:
Done